src/Xamarin.Android.Build.Tasks/Tasks/NdkUtils.cs

```diff
@@ -16,19 +17,28 @@ namespace Xamarin.Android.Tasks
 {
 	public static class NdkUtil
 	{
+		// We need it to work fine during our tests which are executed at the same time in various threads.
```

Comment:
...or we could make our tests *not* run on multiple threads.  I forget how to do that, but I know it's possible.

Reply:
We want the build tests to run in parallel, though... It makes them take much less time. This is a corner case scenario and I think that this solution is perfectly fine here.